spec/features/user_updates_site_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+feature "user updates site", %{
+  As an authenticated user
+  I want to update an item's information
+  So that I can correct errors or provide new information
+  } do
+
+  scenario 'authenticated user sees edit button on show page' do
+    edit_site = FactoryGirl.create(:site)
+    visit new_user_session_path
+
+    fill_in "Email", with: edit_site.user.email
+    fill_in "Password", with: "password"
+
+    click_button "Sign in"
+
+    visit site_path(edit_site)
+
+    expect(page).to have_content "Edit"
+
+  end
+  scenario 'unauthenticated user sees edit button on show page' do
+    edit_site = FactoryGirl.create(:site)
+
+    visit site_path(edit_site)
+
+    expect(page).to have_no_content "Edit"
+
+  end
+  scenario 'authenticated user edits site' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.